big_o/__init__.py

```diff
@@ -5,5 +5,6 @@
     infer_big_o_class,
     measure_execution_time,
 )
+from big_o import reports
```

Comment:
Please add `# noqa` at this end of the line, it's so that the linter does not complain

Reply:
Done